client/settings/notices/legacy-experience-transition/index.js

```diff
@@ -1,5 +1,6 @@
 import { useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
+import '@wordpress/notices';
```

Comment:
This import fixes the "Cannot destructure property..." error mentioned by @diegocurbelo in [this review](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3265#pullrequestreview-2173199500).

The error started to happen on my end just by checking out this branch after some time, so I'm not sure what could have changed.

I ended up importing the entire module and not a member of it since we only needed its side effects, but happy to update it.